### PR TITLE
Submit some Intl.Locale() tests.

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

File: test/intl402/Locale/constructor-properties.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+/*---
+esid: pending
+description: >
+    Checks the properties of the Locale constructor.
+info: |
+    Every other data property described in clauses 18 through 26 and in Annex B.2 has the attributes { [[Writable]]: true, [[Enumerable]]: false, [[Configurable]]: true } unless otherwise specified.
+    Unless specified otherwise in this document, the objects, functions, and constructors described in this standard are subject to the generic requirements and restrictions specified for standard built-in ECMAScript objects in the ECMAScript 2019 Language Specification, 10th edition, clause 17, or successor. 
+    The Locale constructor is a standard built-in property of the Intl object.
+---*/
+
```
Comment (rwaldron) [inline test/intl402/Locale/constructor-properties.js]: @leobalter I'm not going to block review on this file, we can fix it in a follow up

**Comment by Ms2ger** [comment]
@lukewagner should I assign copyright for my code to Mozilla or Igalia?

**Comment by littledan** [comment]
@Ms2ger We've been including Igalia, S.L. in our test262 contributions generally; I'd prefer that here, if that's OK with Mozilla.

**Comment by anba** [comment]
FWIW I'm fine with leaving my name off the copyright line.

**Comment by scott-wagner** [comment]
@Ms2ger Following previous precedent and using Igalia sounds great.

**Comment by lukewagner** [comment]
@Ms2ger Following previous precedent and using Igalia sounds great.

File: features.txt [hunk 2]
```diff
@@ -77,6 +77,10 @@ numeric-separator-literal
 String.prototype.matchAll
 Symbol.matchAll
 
+# Intl.Locale
+# https://github.com/tc39/proposal-intl-locale
+Intl.Locale
```
Comment (rwaldron) [inline features.txt]: Excellent

File: test/intl402/Locale/instance-properties.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+description: >
+    Verifies the properties of a Locale instance.
+info: |
+    Intl.Locale( tag [, options] )
+    6. Let locale be ? OrdinaryCreateFromConstructor(NewTarget, %LocalePrototype%, internalSlotsList).
+features: [Intl.Locale]
+---*/
+
+const value = new Intl.Locale("en");
+assert.sameValue(Object.getPrototypeOf(value), Intl.Locale.prototype);
```
Comment (leobalter) [inline test/intl402/Locale/instance-properties.js]: +1

**Comment by rwaldron** [comment]
Landed

**Pull request closed without merging**